index.md

```diff
@@ -25,34 +25,100 @@ develop the model and apply it to a range of science and policy
-the greenhouse gas consequences of specific policy measures or energy
-technology adoption including; CO2 capture and storage, bioenergy,
-hydrogen systems, nuclear energy, renewable energy technology, and
-energy use technology in buildings, industry and the transportation
-sectors. GCAM is an Representative Concentration Pathway (RCP)-class
-model. This means it can be used to simulate scenarios, policies, and
-emission targets from various sources including the Intergovernmental
-Panel on Climate Change (IPCC). Output includes projections of future
-energy supply and demand and the resulting greenhouse gas emissions,
-radiative forcing and climate effects of 16 greenhouse gases, aerosols
-and short-lived species at 0.5×0.5 degree resolution, contingent on
-assumptions about future population, economy, technology, and climate
-mitigation policy.
-
-## Documentation
+## GCAM Model Overview
+
+### Introduction
+
+GCAM is a global integrated assessment model that represents the
+behavior of, and complex interactions between five systems: the energy
+system, water, agriculture and land use, the economy, and the
+climate. GCAM has been under development for over 30 years. Work began
+in 1980 with the work first documented in 1982 in working papers
+(Edmonds and Reilly, 1982a,b,c) and the first peer-reviewed
+publications in 1983 (Edmonds and Reilly, 1983a,b,c). A this point,
+the model was known as the Edmonds-Reilly (and subsequently the
+Edmonds-Reilly-Barnes) model. The model was renamed MiniCAM in the
+mid-1990s, and the model was renamed to GCAM in the mid-2000s. The
+first coupling to a carbon cycle model was published in Edmonds, et
+al. (1984). The first use of GCAM (MiniCAM at the time) in conjunction
+with a Monte Carlo uncertainty analysis was published in Reilly,
+Edmonds, Gardner and Brenkert (1987).
+
+Throughout its lifetime, GCAM has evolved in response to the need to
+address an expanding set of science and assessment questions. The
+original question that the model was developed to address was the
+magnitude of mid-21st-century global emissions of fossil fuel
+CO2. Over time GCAM has expanded its scope to include a wider set of
+energy producing, transforming, and using technologies, emissions of
+non-CO2 greenhouse gases, agriculture and land use, water supplies and
+demands, and physical Earth systems. It is increasingly being used in
+multi-model, multi-scale analysis, in which it is either soft- or
+hard-coupled to other models with different foci and often greater
+resolution in key sectors. For example, it has been coupled to a state
+of the art Earth system model (Collins, et al., 2015). GCAM has been
+used to produce scenarios for national and international assessments
+ranging from the very first IPCC scenarios (Response Strategies
+Working Group, 1990) through the present Shared Socioeconomic Pathways
+(Ebi, et al., 2014). Hundreds of papers have been published in
```

Comment:
Maybe better to update to the SSP IAM papers (Calvin et al., 2017)

Reply:
Also, do we need to actually add these references?

Reply:
I'll add the GCAM SSP references and clean them up in general. (I do like having the references, provides a bit of backup for the long history of GCAM.)